testproject/tests/test_second_step_authentication.py

```diff
@@ -151,22 +182,21 @@ def test_second_method_activation_already_active(active_user_with_email_otp):
         path="/auth/email/activate/",
         format="json",
     )
-    error_msg = "MFA method already active."
     assert response.status_code == 400
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too